### PR TITLE
Add makefile for docker commands

### PR DESCRIPTION
**Pull request by stevedya**
This PR is for the lazy people like me. It is for the creation of a makefile for the docker compose commands. This will simplify the commands that you need to enter when working with the docker environment. For example running tests becomes 
```
make test
```
which was 
```
docker-compose exec -w /code/wagtail web python runtests.py
```

This PR also updates the readme to use those commands instead of the longer versions of them.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+build: ## Build the backend Docker image
```
Comment (zerolab) [inline Makefile:10]: suggestion: add https://github.com/torchbox/wagtailmedia/blob/main/Makefile#L1-L12 to make this self-documenting

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	docker-compose build web
+
+up: ## Bring the backend Docker container up
```
Comment (zerolab) [inline Makefile]: suggestion: `make start` would be more consistent with `make stop` (start/stop is easier to remember then up/stop)

File: README.md [hunk 3]
```diff
@@ -86,34 +86,76 @@ frontend   docker-entrypoint.sh /bin/ ...   Up
...
+### Run tests for a specific file
+
+```sh
+make test file=wagtail.admin.tests.test_name.py
 ```
 
 ### Open a Django shell session
 
 ```sh
-docker-compose exec web python manage.py shell
```
Comment (zerolab) [inline README.md:98]: perhaps use both commands (make and direct one), in all examples?
some people may prefer the direct command

**Review by zerolab — COMMENTED** [review]
Left a few suggestions. Kudos for not being *that* lazy and contributing these!

File: Makefile (hunk 1 above)
Comment (stevedya) [reply Makefile:10]: Love this! Thank you, i've added now

File: Makefile (hunk 2 above)
Comment (stevedya) [reply Makefile]: Good idea! I've made this change now too

File: README.md (hunk 3 above)
Comment (stevedya) [reply README.md:98]: A good idea for sure, I'm just curious how we would want to document it?
Something like this perhaps?

| Make command | Raw command                |
|--------------|----------------------------|
| `make build` | `docker-compose build web` |

Comment (zerolab) [reply README.md:98]: that's a good format as any!

File: docker-compose.yml [hunk 4]
```diff
@@ -35,15 +35,15 @@ services:
...
+#    build:
+#      context: .
+#      dockerfile: Dockerfile.frontend
+#    working_dir: /code/wagtail
+#    volumes:
+#      - ./wagtail:/code/wagtail:delegated,rw
+#      - node_modules:/code/wagtail/node_modules/
+#    command: bash -c "echo 'Copying node_modules, this may take a few minutes...' && rsync -rah --info=progress2 /node_modules /code/wagtail/ && npm run start"
+#    restart: "no"
+#    tty: true
```
Comment (saevarom) [inline docker-compose.yml]: You have commented out the frontend container in this PR, please add it back :)

File: README.md (hunk 3 above)
Comment (saevarom) [reply README.md:98]: I agree, we should have documentation for both approaches.

File: docker-compose.yml (hunk 4 above)
Comment (zerolab) [reply docker-compose.yml]: Looks like "Add template for dialog component" (4c5e18c) was not really meant for this repot either

Comment (stevedya) [reply docker-compose.yml]: Ooops! I have reverted that mistake commit with the commented FE container now haha. 😄

**Comment by saevarom** [comment]
This looks good, I'll merge it.

**Pull request merged**